packages/components-vue/projects/vue-wrapper/tests/unit/specs/utils.spec.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  addEventListenerToElementRef,
+  getPrefixedTagName,
+  prefixInjectionKey,
+  syncProperties,
+} from '../../../src/utils';
+import * as Vue from 'vue';
+
+describe('getPrefixedTagName()', () => {
+  it('should call inject() with correctParameters', () => {
+    const spy = jest.spyOn(Vue, 'inject').mockReturnValue('');
+
+    getPrefixedTagName('p-text');
+
+    expect(spy).toBeCalledWith(prefixInjectionKey);
+  });
+
+  it('should call throw error if prefix is undefined', () => {
+    jest.spyOn(Vue, 'inject').mockReturnValue(undefined);
+
+    let error;
+    try {
+      getPrefixedTagName('p-text');
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toBeDefined();
+  });
+
+  it('should return tagName if no prefix is defined', () => {
```

Comment:
`tagName` might be misleading, also test description doesn't match what is happening.

Reply:
Added `passed tagName` to clarify that its about the parameter